docs/add-ins/addin-tutorial.md

```diff
@@ -501,15 +596,22 @@ function buildFileList(files) {
 }
 ```
 
-That fully implements the settings dialog. Now the question is how do we invoke it? You may have noticed that we did not add a button to the ribbon for settings. Instead, the add-in will check that it has been configured. If it hasn't, then it will prompt the user when they invoke the add-in to configure before proceeding. Since the user could click either button first, we'll do this check in both cases.
+> [!NOTE]
+> You may have noticed that there's no button to invoke the settings dialog. Instead, the add-in will check whether it has been configured when the user selects either the **Insert default gist** button or the **Insert gist** button. If the add-in has not yet been configured, the settings dialog will prompt the user to configure before proceeding. 
```

Comment:
Thanks; in general I agree that present-tense is better than future-tense. I opted for future tense here because the functionality being described hasn't been implemented yet (it comes later in the tutorial).